### DISCUSSION TITLE
Bus stop not in response despite exact match

### DISCUSSION THREAD

**Discussion by felix-geovelo** [komoot/photon#806] [Q&A] [answered]
I can't figure out why the bus stop "Verdun" in Tours (France) is not in the response of photon when searching for "verdun tours"
see : 
[https://photon.komoot.io/api?q=verdun%20tours](https://photon.komoot.io/api?q=verdun%20tours)
The response only contain 2 elements, there is free space to put the bus stop located in Tours

And yes, it's in the photon index because this is returning it:
[https://photon.komoot.io/api?q=verdun%20tours&osm_tag=highway:bus_stop](https://photon.komoot.io/api?q=verdun%20tours&osm_tag=highway:bus_stop)

I'm aware of the deduplication process but i think i'm not in this case, since the name of the bus stop is "Verdun" and the name in the result of the first request is "Rue de Verdun", they're not the same key and the bus stop shouldn't be evicted.

If anyone can enlighten me :pray: 

Thanks !

**Comment by lonvia** [comment]
Streets are considered of higher importance than simple pois, so they end up with a much higher score. The additional 'full name match' can't really cancel that out. It is a bit tricky to find the right balance here. This is especially true for bus stops which tend to have names that are easily confused with the street/attraction the bus stop is for.

**Reply by felix-geovelo** [reply]
Thanks for the explanation, there is still one thing that is not clear for me:
In the first request, the bus stop is not in the response because his importance is too far (and low) from the importance of the street ? Is it possible to tune that importance difference somewhere in the code ? Like a threshold or something ?

**Reply by lonvia** [reply, marked as answer]
Not really. You can play with the weights that are applied in the search function but be aware this is all a question of trial-and-error.

**Reply by felix-geovelo** [reply]
Ok, thanks for the response !
